content/sensu-go/6.3/web-ui/bsm-module.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Build business services with the business service monitoring (BSM) module"
+linkTitle: "Build Business Services with the BSM Module"
+description: "The Sensu web UI includes a module for creating, configuring, editing, and deleting business service monitoring (BSM) service components and rule templates. Read this page to learn how to use the Sensu web UI BSM module."
+weight: 30
+version: "6.3"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.3:
+    parent: web-ui
+---
+
+**COMMERCIAL FEATURE**: Access business service monitoring (BSM) in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][3].
+
+The Sensu web UI includes a module to help you build and configure business service monitoring (BSM) [service components][1] and [rule templates][2].
+
+## Build a business service
+
+To build a business service in the web UI module:
+
+1. Click the services icon in the left navigation menu to open the Services page: ![services icon](/images/web-ui-services-icon.png)
+2. Click **ADD NEW SERVICE** to open the Create New Service dialog window.
+3. Enter the metadata for the new service and click **Submit**.
+
+The Services page will be updated to include a tile for the new service.
+
+The business service itself is an entity with the class `service`, so it will also be listed on the Entities page.
+
+To add service components to a business service:
+
+1. Click the edit icon for the business service: ![edit icon](/images/web-ui-edit-icon.png)
+2. Select **Add Component** from the drop-down menu to open the Configure New Service Component dialog window.
+3. Enter the metadata, query selectors, rule templates, check scheduling, and handlers for the service component and click **Submit**.
+
+The list on the business service tile will be updated to include the service component. 
+
+## View and manage business services
+
+After you create a business service by any means (web UI, API, or sensuctl), it will be listed in the web UI Services page until you delete it.
+Click the business service name to view its events and other related details.
+You can also edit, silence, and delete the service from the detail page.
+
+To edit, add components to, or delete a business service, click the edit icon in the top-right corner of the service's tile.
```

Comment:
Technically, I think the icon is often referred to as a "kebab", "vertical disclosure", or "more" button.. though, I'm not sure if anyone of those is universally understood.

Reply:
I see though that the web UI is using "kebab" so I will go with that for consistency. Thank you!